content/languages/python/authoring/index.md

```diff
@@ -0,0 +1,225 @@
+---
+kind: recipe
```

Comment:
```suggestion
kind: tutorial
```

Recipe is problem oriented. It should present a problem and provide a solution to it.

This is a tutorial because it's meant to familiarize someone new. It should walk through what's needed to create a Python kata.
I think this document should focus on just the basics and link to further readings. For example, "Accessing the solution file" should be a separate recipe. It's useful for even experienced authors.

See https://github.com/codewars/docs/issues/8#issuecomment-658961005

Reply:
Just as I mentioned in #191, I will try to extract some common and general parts to other articles, when I create more pages like this one. Next ones I am thinking about are JS (because it's popular) and C (because it causes many problems to authors with memory management) or C++.